packages/app/src/app/globals.css

```
@@ -0,0 +1,2646 @@
+  outline: none;
+}
+
+.header-link svg {
+  width: 16px;
+  height: 16px;
+  stroke-width: 2;
+}
+
+.header-link .github-mark,
+.header-link .nervos-mark {
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)